rpc/tests/blobstream.rs

```diff
@@ -0,0 +1,31 @@
+#![cfg(not(target_arch = "wasm32"))]
+
+use celestia_rpc::blobstream::BlobstreamClient;
+
+pub mod utils;
+
+use crate::utils::client::{new_test_client, AuthLevel};
+
+#[tokio::test]
+async fn get_data_root_tuple_root() {
+    let client = new_test_client(AuthLevel::Write).await.unwrap();
+
+    let result = client
+        .get_data_root_tuple_root(4211296, 4211298)
```

Comment:
those tests seem to fail on CI.
```
called `Result::unwrap()` on an `Err` value: Call(ErrorObject { code: ServerError(1), message: "end block 4211300 is higher than local chain height 28. Wait for the node until it syncs up to 4211300", data: None })
```
could you instead make `header_get_network_head()` request and adjust height accordingly?

Reply:
noted!